template_test.go

```diff
@@ -81,6 +81,13 @@ func TestTemplate(t *testing.T) {
 	assert.True(t, opts.ROpts.ThriftDisableEnvelopes)
 }
 
+func TestTemplateRequestsField(t *testing.T) {
+	opts := newOptions()
+	mustReadYAMLFile(t, "testdata/templates/stream.yab", opts)
+
+	assert.Equal(t, "---\ntest: 1\n---\ntest: 2\n", opts.ROpts.RequestJSON)
```

Comment:
nit: Let's have the following format (easier to read) for the test

```
want = `---
test: 1
---
test: 2
```

assert.Equal(t, want, opts.ROpts.RequestJSON)

(off commit) having `opts.ROpts.RequestJSON` and not json as value is odd

Reply:
Yup `RequestJSON` weirdly carries YAML as well. This needs some cleanup for sure